src/platform_impl/windows/window.rs

```diff
@@ -393,6 +393,17 @@ impl Window {
         let window = self.window.clone();
         let window_state = Arc::clone(&self.window_state);
 
+        let fullscreen = match fullscreen {
+            Some(Fullscreen::Borderless(None)) => {
+                let current_monitor = Some(RootMonitorHandle {
+                    inner: monitor::current_monitor(window.0),
+                });
+
+                Some(Fullscreen::Borderless(current_monitor))
+            }
+            _ => fullscreen,
+        };
```

Comment:
Why can't this be stored with None as Borderless? Do you want to make it possible to go from fullscreen on one monitor to fullscreen on another using None as parameter? When would you ever want to do that?

Reply:
>Why can't this be stored with None as Borderless? Do you want to make it possible to go from fullscreen on one monitor to fullscreen on another using None as parameter? When would you ever want to do that?

it's due to monitor logic in event_loop.rs when you can move fullscreren window across the monitor, since I bet the current monitor after move is a new monitor, so you won't be able to update things properly. We may always update monitor there if the current one is `None` though.